### DISCUSSION TITLE
SIGIR 2021 E-Commerce Workshop Data Challenge

### DISCUSSION THREAD

**Discussion by tszumowski** [RUCAIBox/RecBole#816] [Ideas]
The SIGIR 2021 Conference has an e-commerce workshop data challenge, called **In-session prediction for purchase intent and recommendations**. It just opened up the other day.

https://sigir-ecom.github.io/data-task.html

It's been a while since I had a chance to use recbole due to other professional activities. However, in my experience from a few months ago, I feel as if recbole would be a good fit for this dataset. The input formats are similar, and the desired algorithms are in the space of recbole. 

It may be worth participating by benchmarking recbole's capabilities against the dataset. It can provide additional recbole awareness to a potentially new community. 

Unfortunately I can't commit any time to it, but figured I'd pass along the info here in case you or anyone else is interested.

**Comment by 2017pxy** [comment]
@tszumowski  Thanks for the information, we will keep an eye on it.
